### ISSUE TITLE
UI Feedback when resending codes in auth in-app

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#3212]
In the unblock code and session email code views, we have a 'resend code' button. If the user clicks on it, it would be nice to have some graphical effect to communicate that something has been done.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-2006)
